Server/trunk/sqlite-3.6.23.1/CMakeLists.txt

```diff
@@ -21,12 +21,12 @@ include_directories("${NPLRUNTIME_ROOT}/Core")
 # include boost if client is not defined, it allows us to use a different boost version than the client on the server build.
 if (PARAENGINE_CLIENT OR WIN32)
```

Comment:
this entire code block can be removed from cmake. since both client and server share the same boost